### PR TITLE
React icons

### PR DESCRIPTION
**Pull request by Kevin-Solomon**
# Bugs Fixed

**Problem Statement :**
FontAwesome was returning a 404 response while fetching icons to display on the UI.

Fix : 
Used React Icons.
Made ```getIcons``` function to fetch different icons based on parameters.

### CODE REVIEW

File: src/components/Card/Card.css [hunk 1]
```diff
@@ -132,5 +132,11 @@
   border: 1px solid var(--primary-color);
 }
 .icon-btn {
-  visibility: hidden;
+  /* visibility: hidden; */
```
Comment (aamir74) [inline src/components/Card/Card.css]: Remove comments if not required

**Review by aamir74 — APPROVED** [review]
Well written formatted code.
Icons are visible.
UI looks good.

**Review by RakshaPawar108 — COMMENTED** [review]
The icons look great Kevin! Just a question: In single product page, if I click on Add to Cart button, it gets redirected to a blank page. I am not sure if you have added the functionality to add to cart from the single product page, so is it a feature yet to be added or is it a bug?

**Comment by Kevin-Solomon** [comment]
> The icons look great Kevin! Just a question: In single product page, if I click on Add to Cart button, it gets redirected to a blank page. I am not sure if you have added the functionality to add to cart from the single product page, so is it a feature yet to be added or is it a bug?

 I have yet to add to cart functionality on single product page, thank you for pointing it out Raksha ❤️.
 I will solve this bug in the next PR.

**Review by RakshaPawar108 — APPROVED** [review]

**Pull request merged**